ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/testUserExtraManifest/01-endUser-ex-machine-config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: load-endusr-mc
```

Comment:
I see this file being added, and the directory referenced from the SiteConfig, but nothing checks that it was actually added to the final result.